src/library_eventloop.js

```diff
@@ -52,10 +52,8 @@ LibraryJSEventLoop = {
     setImmediateWrapped.mapping[id] = undefined;
   },
 
-  $polyfillSetImmediate__deps: ['$setImmediateWrapped', '$clearImmediateWrapped'],
-  $polyfillSetImmediate__postset: `
-    var emSetImmediate;
-    var emClearImmediate;
+  $emSetImmediate__deps: ['$setImmediateWrapped', '$clearImmediateWrapped', '$emClearImmediate'],
+  $emSetImmediate__postset: `
```

Comment:
Does this need to be in a postset and then to set the actual value to undefined? I seem to remember we can write a string value to the key (and if it is multiline, it is the actual code).

Reply:
This code actually defines two symbols emSetImmediate and emClearImmediate.  Its possible that we could write this as an IFFE that returns `emSetImmediate` while setting `emClearImmediate` as a side effect?  That would be a bigger change though.  This approach seems reasonable to and slightly cleaner than before.